### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by 4Sol** [Bionus/imgbrd-grabber#2707]
When after downloading the software I launched it, I got and error: No source founded

**Comment by Bionus** [comment]
Hello, you'll need to give a little bit more details here 😅 
What version are you trying to install, and on which platform? Is it via installer, AppImage, ZIP, tar.gz, etc.?

**Comment by gabrielwoj** [comment]
Hello. I am having the same issue. I decided to download the Portable version due to the installer having 3 results on VirusTotal (which are most likely false positives, but I still prefer using portable programs whenever possible).

I tried to use the x64 version for Grabber 7.9.0 (x64). I use Windows 10 Pro 21H1. Tried on the x86 version and didn't work it either.

Log from main.log (x86):

`[23:00:34.086][Error] JavaScript helper file could not be opened`
`[23:00:34.089][Info] Temp directory purged of 0/0 files (0 failed)`
`[23:00:44.710][Info] Software version: 7.9.0.`
`[23:00:44.711][Info] Software CPU architecture: x86.`
`[23:00:44.711][Info] Computer CPU architecture: x86_64.`
`[23:00:44.711][Info] Qt CPU architecture: i386.`
`[23:00:44.712][Info] Computer platform: Windows 10 Version 2009.`
`[23:00:44.712][Info] Path: "D:/Portable Program Files/Art Related/Grabber_v7.9.0_x86"`
`[23:00:44.713][Info] Loading preferences from "D:/Portable Program Files/Art Related/Grabber_v7.9.0_x86/settings.ini"`
`[23:00:44.713][Info] Temporary path: "C:/Users/[Username_Redacted]/AppData/Local/Temp\Grabber"`
`[23:00:44.713][Info] Sources found: 0`
`[23:00:44.714][Info] SSL libraries: OpenSSL 1.1.1o  3 May 2022`
`[23:00:44.808][Info] Setting language to 'English' ("D:\Portable Program Files\Art Related\Grabber_v7.9.0_x86\languages\English.qm")...`

P.S.: My username folder has no special characters on it. I also had to change the path to include quotations otherwise it would break Github's code template.

The x64 is basically the same, just replacing x86 with x64, though the SSL library is "OpenSSL 1.1.1d  10 Sep 2019" on x64.

Thanks.

**Comment by Bionus** [comment]
Indeed, it would seem that the ZIP version doesn't have the "sites" directory bundled with it.
7.9.0 was the first version to introduce it and it seems I overlooked that.
